sample-apps/delivery-customizations-sample-app/web/index.js

```diff
@@ -344,18 +359,35 @@ export async function createServer(
               ...METAFIELD,
               ownerId: gid,
               type: "single_line_text_field",
-              value: payload.deliveryOptionName,
+              value: deliveryOptionName,
             },
           ],
         },
       },
     };
 
+    let reducer = ({ metafieldsSet }) => {
+      return {
+        metafields: metafieldsSet?.metafields[0],
+        userErrors: metafieldsSet?.userErrors
+      }
+    };
+
     const { status: metafieldStatus, data: metafieldData } =
-      await queryResponse(req, res, query);
+      await queryResponse(req, res, query, reducer);
 
-    if (metafieldStatus !== 200)
+    if (metafieldStatus !== 200 ){
```

Comment:
Nit: 
```suggestion
    if (metafieldStatus !== 200) {
```

We should probably just run prettier before shipping 😅

Reply:
Will do!